packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -51,7 +51,7 @@ dependencies: {{#enable_dev_menu}}
   widget_toolkit: ^0.2.0 {{#enable_pin_code}}
   widget_toolkit_biometrics: ^0.1.0 {{/enable_pin_code}} {{#enable_feature_otp}}
   widget_toolkit_otp: ^0.1.0{{/enable_feature_otp}}{{#enable_pin_code}}
-  widget_toolkit_pin: ^0.2.0{{/enable_pin_code}}
+  widget_toolkit_pin: 0.2.0{{/enable_pin_code}} # TODO: Temporary fix until widget_toolkit_pin version >0.2.1 is released and lib_pin tests pass
```

Comment:
We don't need to introduce temporary fixes. As mentioned in the other comment @GoranPrime / @DDavidPrime  needs to fix and distribute a new version of the widget_toolkit_pin where the mentioned issue is fixed

Reply:
Returned it